src/media/css/site.styl

```diff
@@ -284,7 +284,11 @@ a:active {
     }
     section.full > div {
         padding: 30px 0;
-        width: $max-site-width;
+        max-width: $max-site-width;
+    }
+    .placeholder + .placeholder section.full,
```

Comment:
That's fine for widths near 1050px, but it inadvertently affects wider desktops:

### PR 

![screen shot 2015-02-17 at 9 54 11 pm](https://cloud.githubusercontent.com/assets/674727/6243082/98e4161c-b6ef-11e4-8439-d501ace5f333.png)

### Currently on dev

![screen shot 2015-02-17 at 9 54 14 pm](https://cloud.githubusercontent.com/assets/674727/6243083/9acdfa1a-b6ef-11e4-9903-9ea096016d37.png)